src/main/java/com/revature/services/impl/DistanceServiceImpl.java

```diff
@@ -146,10 +155,14 @@ public String getGoogleMAPKey() {
         Map<String, String> env = System.getenv();
         for (Map.Entry <String, String> entry: env.entrySet()) {
             if(entry.getKey().equals("googleMapAPIKey")) {
-                return entry.getValue();
+                logger.warn("entry was a good map API key");
+            	return entry.getValue();
+            	
             }
         }
+        logger.warn("entry was not a google map API key");	
```

Comment:
If this is telling us that we're using an invalid key then it should probably be an ERROR log. We should probably check if it is empty as well - if it's empty WARN of no key available, if key exists but is invalid provide an ERROR